### PR TITLE
Making REST Api callout using LWC component

### PR DESCRIPTION
**Pull request by ManishKumarGupta8972**
Ticket included : [AVE_BLOG90](https://theavenoir.atlassian.net/browse/AVEBLOG-109)
Issue : Making REST API callout using LWC component
Cause : N/A
fix: N/A

### CODE REVIEW

**Comment by divyaKhare07** [comment]
API should be in all Uppercase in the Issue

File: force-app/main/default/lwc/dogImagesUsingFetchApi/dogImagesUsingFetchApi.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Created by: Manish Kumar Gupta
+Ticket Number: AVEBLOG-109
+------------------------------------------------------------------------*
+Blog: Making REST Api callout using LWC component
+------------------------------------------------------------------------*
+Version History:*
+VERSION    DEVELOPER NAME        DATE         DETAIL FEATURES
+1.0       Manish Kumar Gupta     02/10/2024   Initial Development
+***********************************************************************/
+import { LightningElement } from 'lwc';
```
Comment (AvenoirShiv) [inline force-app/main/default/lwc/dogImagesUsingFetchApi/dogImagesUsingFetchApi.js:12]: remove the space before and after the curly brackets

File: force-app/main/default/lwc/dogImagesUsingFetchApi/dogImagesUsingFetchApi.css
Comment (AvenoirShiv) [inline force-app/main/default/lwc/dogImagesUsingFetchApi/dogImagesUsingFetchApi.css:1]: You are using custom labels in you codes but those are misssing in your pr. Please includes all the custom lables in pr too

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 2]
```diff
@@ -637,4 +637,60 @@
         <shortDescription>Record Created Successfully</shortDescription>
         <value>Record Created Successfully</value>
     </labels>
+    <labels>
+        <fullName>Dog_Api_Url</fullName>
```
Comment (AvenoirShiv) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:641]: Full name should be in Uppercase seperated by underscore eg  SELECT_SUB_GENRES

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 3]
```diff
@@ -637,4 +637,60 @@
...
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Dog Api Url</shortDescription>
+        <value>https://dog.ceo/api/breeds/image/random</value>
+    </labels>
+    <labels>
+        <fullName>Dog_Image_Generator_Button_Label</fullName>
```
Comment (AvenoirShiv) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:648]: Full name should be in Uppercase seperated by underscore eg  SELECT_SUB_GENRES

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 4]
```diff
@@ -637,4 +637,60 @@
...
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Dog Image Generator Button Label</shortDescription>
+        <value>🐶Generate Random Dog Picture</value>
+    </labels>
+    <labels>
+        <fullName>Dog_Image_Generator_Title</fullName>
```
Comment (AvenoirShiv) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:655]: Full name should be in Uppercase seperated by underscore eg  SELECT_SUB_GENRES

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 5]
```diff
@@ -637,4 +637,60 @@
...
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Dog Image Generator Title</shortDescription>
+        <value>Random Dog Picture Generator</value>
+    </labels>
+    <labels>
+        <fullName>Error_Message_For_HttpError</fullName>
```
Comment (AvenoirShiv) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:662]: Full name should be in Uppercase seperated by underscore eg  SELECT_SUB_GENRES

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 6]
```diff
@@ -637,4 +637,60 @@
...
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error Message For HttpError</shortDescription>
+        <value>The server responded with an error. Please try again later.</value>
+    </labels>
+    <labels>
+        <fullName>Error_Message_For_TypeError</fullName>
```
Comment (AvenoirShiv) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:669]: Full name should be in Uppercase seperated by underscore eg  SELECT_SUB_GENRES

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 7]
```diff
@@ -637,4 +637,60 @@
...
+    </labels>
+    <labels>
+        <fullName>Error_Message_Title_Fetch_Api</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error Message Title Fetch Api</shortDescription>
+        <value>Error Fetching Dog Image</value>
+    </labels>
+    <labels>
+        <fullName>HTTP_Label</fullName>
```
Comment (AvenoirShiv) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:690]: Full name should be in Uppercase seperated by underscore eg  SELECT_SUB_GENRES

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 8]
```diff
@@ -637,4 +637,60 @@
...
+    </labels>
+    <labels>
+        <fullName>Error_Message_For_UnexpectedError</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error Message For UnexpectedError</shortDescription>
+        <value>An unexpected error occurred. Please try again.</value>
+    </labels>
+    <labels>
+        <fullName>Error_Message_Title_Fetch_Api</fullName>
```
Comment (AvenoirShiv) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:683]: Full name should be in Uppercase seperated by underscore eg  SELECT_SUB_GENRES

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 9]
```diff
@@ -637,4 +637,60 @@
...
+    </labels>
+    <labels>
+        <fullName>Error_Message_For_TypeError</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error Message For TypeError</shortDescription>
+        <value>There was a network issue. Please check your internet connection.</value>
+    </labels>
+    <labels>
+        <fullName>Error_Message_For_UnexpectedError</fullName>
```
Comment (AvenoirShiv) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:676]: Full name should be in Uppercase seperated by underscore eg  SELECT_SUB_GENRES

**Review by AvenoirShiv — APPROVED** [review]
Looks Fine

**Pull request merged**